package.json

```diff
@@ -23,6 +23,7 @@
     "@emotion/styled": "^11.3.0",
     "@sentry-internal/global-search": "^0.0.43",
     "@sentry/integrations": "6.8.0",
+    "@sentry/node": "6.8.0",
```

Comment:
See my earlier comment in `sentry-instrumentation`

Reply:
141.4 kB MINIFIED
41.3 kB MINIFIED + GZIPPED

I'm not sure if it's worth it just for a type, maybe it's better to use any

Reply:
This doesn't get packaged into anything so I think we're fine here. It won't add to our bundle size.

Reply:
I see.. so it's fine then